src/django_components_js/README.md

```diff
@@ -0,0 +1,62 @@
+# Django componnets JS
+
+## Usage
+
+```js
+// Register a function that is run at component initialization
+Components.manager.registerComponent(
+  "table",  // Component name
+  async (data, { id, name, els }) => {
```

Comment:
I updated the API of the JS manager code so that it aligns with when we add the `$onLoad` method.